### PR TITLE
[Fix] CLI broken with redis errors and hanging

### PR DESCRIPTION
**Pull request by adrinr**
## Description
The new doc-writethrough class is declared in backend-core, and the queue gets initialised by default. We are having issues with CLI, as it tries to initialise this queue (even if it's not needed) but the redis connection might not exist.

image

Updating this class in order to create the queue when actually required.

## Addresses
- Linear: [BUDI-8100 - Budi CLI Broken with Redis Errors and Hanging
](https://linear.app/budibase/issue/BUDI-8100/budi-cli-broken-with-redis-errors-and-hanging)
- https://github.com/Budibase/budibase/issues/13276

## Launchcontrol
Fix CLI issue with the doc-writethrough initialisation

### CODE REVIEW

File: packages/cli/tsconfig.build.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "types": ["node", "jest"],
     "outDir": "dist",
     "skipLibCheck": true,
+    "baseUrl": ".",
```
Comment (mike12345567) [inline packages/cli/tsconfig.build.json:14]: Why is this needed out of interest?

**Review by mike12345567 — APPROVED** [review]

File: packages/cli/tsconfig.build.json (hunk 1 above)
Comment (adrinr) [reply packages/cli/tsconfig.build.json:14]: This is needed in order to properly use the tsconfig-paths: 
image
